router_types.c

```diff
@@ -44,6 +47,55 @@ static const struct router_type *router_types[] = {
 	NULL,
 };
 
+int read_mac(uint8_t mac[ETH_ALEN], const char *macstr)
+{
+	int ret;
+
+	ret = sscanf(macstr, "%02hhX:%02hhX:%02hhX:%02hhX:%02hhX:%02hhX",
```

Comment:
You now need to add following to the CPPFLAGS (in the Makefile):

    -D__USE_MINGW_ANSI_STDIO=1